docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+       "    <tr>\n",
+       "      <th>Dimensionality reduction before fitting (reduce_dim)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction method (dim_reducer)</th>\n",
+       "      <td>PCA</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Cross-validation strategy (cross_validator)</th>\n",
+       "      <td>KFold</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># parallel jobs (n_jobs)</th>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b66720270343441a80e97f0a6b57d642",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Initializing:   0%|          | 0/2 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<p>Average cross-validation scores:</p>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>model</th>\n",
+       "      <th>short</th>\n",
+       "      <th>r2</th>\n",
+       "      <th>rmse</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>GradientBoosting</td>\n",
+       "      <td>gb</td>\n",
+       "      <td>0.9077</td>\n",
+       "      <td>50.5277</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>RandomForest</td>\n",
+       "      <td>rf</td>\n",
+       "      <td>0.6085</td>\n",
+       "      <td>103.7577</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time taken: 13.609801292419434 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "em.setup(X, y, model_subset=[\"rf\", \"gb\"], param_search=True, param_search_iters=10, \n",
+    "         n_jobs=5, cross_validator=KFold(n_splits=3))\n",
+    "em.compare()\n",
+    "em.print_results()\n",
+    "\n",
+    "end = time.time()\n",
+    "print(f\"Time taken: {end - start} seconds\")"
+   ]
```

Comment:
could add a section "Additional Information" with the version of autoemulate

Reply:
yep, good idea